docs/learning/howto/acl_basic_examples.md

```diff
@@ -0,0 +1,393 @@
+# ACL Basic Concepts with Examples
```

Comment:
I don't see a sidebar-menus.js entry for this file.  It won't show up in the nav menus if it's not added there.